src/middlewared/middlewared/plugins/security/update.py

```diff
@@ -1,10 +1,12 @@
 import middlewared.sqlalchemy as sa
 
 from middlewared.plugins.failover_.disabled_reasons import DisabledReasonsEnum
-from middlewared.plugins.failover_.reboot import FIPS_KEY
 from middlewared.schema import accepts, Bool, Dict, Int, Patch
 from middlewared.service import ConfigService, ValidationError, job, private
 
+FIPS_REBOOT_CODE = 'FIPS'
```

Comment:
I'd rather create an enum class in the `system/reboot_reasons.py` file (or similar) and use that instead of having random global variables in different plugins. This will become important for tools like `hactl`.